packages/amazon-bedrock/src/bedrock-chat-language-model.ts

```diff
@@ -659,3 +641,18 @@ const BedrockStreamSchema = z.object({
   throttlingException: z.record(z.unknown()).nullish(),
   validationException: z.record(z.unknown()).nullish(),
 });
+
+const bedrockProviderOptionsSchema = z.object({
+  additionalModelRequestFields: z.record(z.any()).optional(),
+  reasoningConfig: z
+    .object({
+      type: z.union([z.literal('enabled'), z.literal('disabled')]).nullish(),
+      budget_tokens: z.number().nullish(),
```

Comment:
rm this one - it's been there bc we made a mistake, can now remove (pls add 2nd changeset)